src/assets/css/style.css

```diff
@@ -46,31 +46,13 @@ nav {
   }
 }
 
-.quote-text {
-  clear: both;
-  justify-content: center;
-  line-height: 1.2;
-  font-weight: 500;
-  font-size: 1.5em;
-  margin-top: 2em;
-  margin-right: 0.4em;
-  margin-left: 0.4em;
-}
-
-.quote-author {
-  height: auto;
-  clear: both;
-  font-size: 1em;
-  text-align: right;
-  margin-bottom: 5em;
-}
-
```

Comment:
This shouldn't have been deleted.

Reply:
The code made the content of the quotes unresponsive.  I used respective bootstrap classes that performs same task but are responsive in nature.

Reply:
Hmm, ok. But the general approach should have been to wrap the element with another element and apply CSS classes to that wrapper element. By doing so, we won't have to remove the custom CSS, also at the same time responsiveness of the component would have been achieved.

Reply:
Sure will keep that in mind.